datacenter/dtr/2.2/guides/admin/configure/job-runner.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Jobrunner
+description: Learn about the inner-workings of the jobrunner container in the DTR workflow.
+keywords: docker, job, runner
+---
+
+The jobrunner is presented as a container in DTR. It is a DTR mechanism that:
+
+1. Consumes jobs from a universal jobs queue 
+2. Performs the work of the given action
+
+Having multiple replicas in your cluster increases the number of jobrunner 
+containers, which will increase the number of concurrent jobs.
+
+## Jobrunner Workflow
+The workflow of the jobrunner can be depicted as follows:
+
+[//]: # (Placeholder for jobrunner diagram. @sarahpark will work on the diagram)
+
+When a particular action is performed (see [Job Actions](#job-actions) below) the 
+respective service generates a job that is put onto the jobs queue with an initial 
+status of `waiting`. When a jobrunner worker is available to pick up the action, 
+it will claim the job (i.e: the workerID will be set to the replica ID of the jobrunner
```

Comment:
I think either using `workerID` and `replicaID` or with spaces between them works, but not one and then the other.